client/db.go

```diff
@@ -85,6 +85,18 @@ type DB interface {
 	//
 	// It is likely unwise to call this on a large database instance.
 	PrintDump(ctx context.Context) error
+
+	// AddPolicy adds policy to acp module, if the acp module exists.
+	//
+	// If policy was successfully added to the acp module then a policyID is returned,
+	// otherwise if acp module was not found then returns the following error:
+	// `client.ErrPolicyAddFailureACPModuleNotFound`
```

Comment:
suggestion: If you use square brackets instead of backticks the type should be linked in godocs: `[client.ErrPolicyAddFailureACPModuleNotFound]`

Reply:
Ah nice suggestion, will do.

- [ ] Use square brackets like [client.ErrPolicyAddFailureACPModuleNotFound]